pkg/account/storage/v2/api_key.go

```diff
@@ -94,13 +94,13 @@ func (s *accountStorage) UpdateAPIKey(ctx context.Context, k *domain.APIKey, env
 	return nil
 }
 
-func (s *accountStorage) GetAPIKey(ctx context.Context, id, environmentID string) (*domain.APIKey, error) {
+func (s *accountStorage) GetAPIKey(ctx context.Context, token, environmentID string) (*domain.APIKey, error) {
```

Comment:
The word token could be a bit confusing.
We have the API key and Service Token, but here, we are using the API key.
WDYT?

Reply:
I see, because an object apiKey already declared inside the function so I use the word token but yeah it's a little confuse.
And on second thought, I'll reverse this function as it is and implement a new get api key by apiKey